ena-submission/scripts/create_assembly.py

```diff
@@ -140,9 +141,11 @@ def create_manifest_object(
         try:
             group_info = get_group_info(config, project_table_entry["group_id"])[0]["group"]
             address = group_info["address"]
-            address_string = (f'{address.get("line1", "")}, {address.get("line2", "")}, '
+            address_string = (
+                f'{address.get("line1", "")}, {address.get("line2", "")}, '
```

Comment:
We probably actually want to add the institute name more prominently here - I don't think people actually care about the post code and address, at least that's what I saw when looking at what others submit.

Something like:
`Biozentrum, University of Basel` is more human friendly than `
Klingelbergstrasse 70, 4056 Basel, Switzerland` - what do you think? I know this is unrelated to this PR, just noticed - can make an issue.

Reply:
I changed this in another PR: https://github.com/loculus-project/loculus/pull/2982/files